From 0e7f3da19f1ba94b25b17fec754c0bc96443a0e4 Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Sat, 10 Aug 2013 14:53:50 -0700
Subject: [PATCH] Revert "msm8660: prevent input routing during VOIP recording"

Microphone wasn't working in apps other than Phone. Credit to Johnnyslt for this fix.
---
 msm8660/AudioHardware.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/msm8660/AudioHardware.cpp b/msm8660/AudioHardware.cpp
index f2b578c..9d0d7a5 100644
--- a/msm8660/AudioHardware.cpp
+++ b/msm8660/AudioHardware.cpp
@@ -2570,9 +2570,7 @@ status_t AudioHardware::doRouting(AudioStreamInMSM8x60 *input, uint32_t outputDe
            || (inputDevice == AUDIO_DEVICE_IN_FM_RX)
            || (inputDevice == AUDIO_DEVICE_IN_FM_RX_A2DP)
 #endif
-           || (inputDevice == AUDIO_DEVICE_IN_COMMUNICATION)
         )
-            ALOGV("Ignoring routing for FM/INCALL/VOIP recording");
             return NO_ERROR;
         if (inputDevice != 0) {
             if (inputDevice & AUDIO_DEVICE_IN_BLUETOOTH_SCO_HEADSET) {
-- 
1.8.1.6

